# Silence unexpected errors. Issues known by upstream:
# * Tag tests producing unexpected test suite failures as expected.
# * Remove language causing an autoload error.

--- a/tests/automated/mule-tests.el
+++ b/tests/automated/mule-tests.el
@@ -830,6 +830,7 @@
              tibetan vietnamese-viscii-lower))
 	  (coding-system-for-read 'iso-2022-7))
       (insert-file-contents (locate-data-file "HELLO"))
+      (Known-Bug-Expect-Failure
       (Assert (equal 
        ;; The sort is to make the algorithm of charsets-in-region
        ;; irrelevant.
@@ -838,6 +839,8 @@
                       :test 'string-match :key 'symbol-name)
 	     #'string<)
        sorted-charsets-in-HELLO))
+      )
+      (Known-Bug-Expect-Failure
       (Assert (equal 
        (sort (remove* "^jit-ucs-charset-" (charsets-in-string
                                            (buffer-substring (point-min)
@@ -845,12 +848,13 @@
                       :test 'string-match :key 'symbol-name)
 	     #'string<)
        sorted-charsets-in-HELLO))))
+    )
 
   ;;---------------------------------------------------------------
   ;; Language environments, and whether the specified values are sane.
   ;;---------------------------------------------------------------
   (loop
-    for language in (mapcar #'car language-info-alist)
+    for language in (remove "Chinese-GB" (mapcar #'car language-info-alist))
     with language-input-method = nil
     with native-coding-system = nil
     with original-language-environment = current-language-environment
